### DISCUSSION TITLE
Remove solo instrument

### DISCUSSION THREAD

**Discussion by MarkGuit** [Anjok07/ultimatevocalremovergui#1415] [Q&A]
I realize that UVR is primarily designed to remove vocals from a recording.
I am hoping it is also able to remove a solo instrument from a recording.
If it can do that can someone please tell me the steps to achieving this?

**Comment by JestAK** [comment]
UVR is separating vocal and instrumental too, so you can pick whatever you want)
By default it creates 2 files with solo vocals and solo instrumentals, but in UVR you can tick Vocals Only or Instrumental Only

**Reply by MarkGuit** [reply]
Thanks Jest for your reply. I would love to know **the secret** to separating instruments from the mix.
In particular I would like to separate guitar solo parts and other instruments such as organ etc.
I have tried all kinds of configurations but none of them worked.

**Reply by SilSinn9801** [reply]
Have you tried using the **Demucs** method with the **v4 | htdemucs_6s** model? That method/model combo outputs six stems (vocals, drums, bass, guitar, piano, other); assuming it works well with your desired song, the organ would end up in the stem called “other”.

**Comment by LukeHebert** [comment]
Agreed, would love to know how to remove e.g. a flute track from e.g. a guitar track

**Reply by SilSinn9801** [reply]
Using **htdemucs_6s** model (which outputs six stems: vocals, drums, bass, guitar, piano, & other), very likely the flute would end up in “other” (possibly mixed with any other instruments present there). Now, to isolate that flute from all those other non-drum, non-bass, non-guitar, non-piano instruments, maybe try method **VR Architecture** with model **17_HP-Wind_Inst-UVR**?

**Comment by rschader** [comment]
Is there an option where you can tell UVR how many tracks there should be?

**Reply by JestAK** [reply]
As I know there is some models for 2-stems, 4-stems and 6 stems only

**Comment by loaykhalifa** [comment]
I know There is a Model you can separate Saxophone or wind instrument called
"17_HP-Wind_Inst-UVR"
